contracts/staking/ServiceStakingBase.sol

```diff
@@ -13,11 +13,6 @@ interface IMultisig {
 
 // Service Registry interface
 interface IService {
-    enum UnitType {
-        Component,
-        Agent
-    }
-
```

Comment:
We can't use the `getUnitIdsOfService()` function as it's available only for L1. Instead, we need to get the full service data and extract `agentIds` from it.